projects/igniteui-angular/src/lib/expansion-panel/expansion-panel-header.component.ts

```diff
@@ -173,7 +173,7 @@ export class IgxExpansionPanelHeaderComponent {
     public set disabled(val: boolean) {
         this._disabled = val;
         if (val) {
-            this.tabIndex = -1;
+            this.tabIndex = null;
```

Comment:
```suggestion
            delete this.tabIndex;
```
it is a little better like this 😄